### DISCUSSION TITLE
Compatibility with Flux model ?

### DISCUSSION THREAD

**Discussion by iacchos-fr** [JuergenLeber/SHIFTR#11] [Q&A] [answered]
Hi @JuergenLeber 
A friend of mine wants me to install a SHITR for him. He has a Flux smart trainer (v2 I think). Is the SHIFTR ‘theoretically’ compatible (I understood that FE-C over BLE is supported on Flux)?
I've noticed that the power supply is different from my Neo 2T: I'm simply going to offer him a USB / 5.5-2.5mm cable to be powered with a simple USB charger... Apart from that?

**Comment by JuergenLeber** [comment, marked as answer]
Hi @iacchos-fr!
According to the Garmin Homepage the FLUX also supports FE-C via BLE so it should work. If not I'm sure we can make it work. Just make sure it has the latest firmware installed - that always helps.
As far as I can read the FLUX has a built-in power supply. You can either build the same device as you have an connect a power supply with a suitable connector and a minimum of around 12V to it (the buck converter needs a voltage difference of 4-5V and those 12V power adapters mostly come with routers and switches and should be available cheap) or just go with an USB charger. Then please don't take the cheapest one as they often make problems and interfere with wifi and bluetooth.
